pyproject.toml

```
@@ -19,7 +19,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-great-expectations = "*" # Needs to be ^0.18.4 but keeping unbounded to deal with dependency resolver conflicts
+great-expectations = "0.18.13"
```

```suggestion
great-expectations = "^0.18.13"
```
It's worth gaming this out but I think this should just be at least `0.18.13`.
The lockfile ensures it's always pinned in the container.